content/topics/design/_index.md

```diff
@@ -14,6 +14,13 @@ aliases:
   - /topics/draft-web-design-standards/
   - /topics/prototype/
   - /topics/public-sans/
+  - /2017/11/14/uswds-digitalgov/
+  - /2017/03/2017-03-13-u-s-web-design-standards-releases-version-1-0/
+  - /2015/09/2015-09-28-introducing-the-u-s-web-design-standards/
+  - /2015/11/2015-11-17-gsa-gov-refreshed-with-eye-toward-mobile-users/
+  - /2015/10/2015-10-05-the-content-corner-content-management-system-considerations/
+  - /2015/11/2015-11-10-trends-on-tuesday-5-tips-for-designing-touch-interactions/
```

Comment:
@bonnieAcameron 

We should follow the pathing convention like this: `YYYY/MM/DD/slug`.
Using the filename after the date will cause a build error.
Don't forget to update any `{{ ref }}` shortcodes that link to these files.

Reply:
This is resolved